compose-builder/gen_compose_ext.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
```

Comment:
How about changing the name slightly to be more explicit that it is for secure extension generation? Like `gen_secure_compose_ext.sh`

Reply:
sure

Reply:
renamed